modules/monitoring-setting-up-pod-topology-spread-constraints-for-alertmanager.adoc

```diff
@@ -0,0 +1,70 @@
+Doing so helps ensure that Alertmanager pods are highly available and run more efficiently because workloads are spread across nodes in different data centers or hierarchical infrastructure zones.
+
+You configure pod topology spread constraints for core platform monitoring in the `cluster-monitoring-config` config map.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have created the `cluster-monitoring-config` `ConfigMap` object.
+
+.Procedure
+
+. Edit the `cluster-monitoring-config` `ConfigMap` object in the `openshift-monitoring` namespace:
++
+[source,terminal]
+----
+$ oc -n openshift-monitoring edit configmap cluster-monitoring-config
+----
+
+. Add values for the following settings under `data/config.yaml/alertmanagermain` to configure topology spread constraints:
++
+[source,yaml]
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: cluster-monitoring-config
+  namespace: openshift-monitoring
+data:
+  config.yaml: |
+    alertmanagerMain:
+      topologySpreadConstraints:
+      - maxSkew: 1 <1>
+        topologyKey: monitoring <2>
+        whenUnsatisfiable: DoNotSchedule <3>
+        labelSelector: 
+          matchLabels: <4>
+            app.kubernetes.io/name=alertmanager
+----
+<1> Specify a numeric value for `maxSkew`, which defines the degree to which pods can be unevenly distributed. 
+This field is required, and the value must be greater than zero. 
+The value specified has a different effect depending on what value you specify for `whenUnsatisfiable`.
+<2> Specify a key of node labels for `topologyKey`.
+This field is required.
+Nodes that have a label with this key and identical values are considered to be in the same topology.
+The scheduler will try to put a balanced number of pods into each domain.
+The value `non-exist` indicates that a node label does not exist and that therefore the pods can be scheduled to any node, whether or not the node has the `monitoring` label.
+<3> Specify a value for `whenUnsatisfiable`.
+This field is required.
+Available options are `DoNotSchedule` and `ScheduleAnyway`.
+Specify `DoNotSchedule` if you want the `maxSkew` value to define the maximum difference allowed between the number of matching pods in the target topology and the global minimum. 
+Specify `ScheduleAnyway` if you want the scheduler to still schedule the pod but to give higher priority to nodes that might reduce the skew.
+<4> Specify a value for `matchLabels`. This value is used to identify the set of matching pods to which to apply the constraints.
+
+. Save the file to apply the changes automatically.
++
+[WARNING]
+====
+When you save changes to a `cluster-monitoring-config` config map, the pods and other resources in the `openshift-monitoring` project might be redeployed. 
```

Comment:
When you save changes to a `cluster-monitoring-config` config map

Reply:
fixed